### ISSUE TITLE
Any NOMP is active?

### ISSUE THREAD

**Issue by shtse8** [krisklosterman/node-open-mining-portal#5]
I found the https://github.com/zone117x/node-open-mining-portal is inactive already. No one is solving issue. I would like to know if this fork is still active?

**Comment by ghost** [comment]
@shtse8 
Hi. Some people works under code of NOMP and stratum. What did You whant? Help or use?

**Comment by shtse8** [comment]
At the moment, use. Because I just setup MPOS and findinga stratum for using. But it is very hard to find one to use. all are inactive and many problems exist. many issues are unresolved. `NOMP` needs a lower version npm to update. `Stratum Mining` can only work for scrypt algo. `CoiniumServ` is wired, some clients cannot connect to my stratum and get many rejects.

**Comment by krisklosterman** [comment]
If you use this fork you will need to update NODE to version 8.x+
I can say that the scrypt and scrypt-n algos are production ready. I have not gotten to test the others yet so I can not say what state they are in.
I plan to continue to work on this fork, but currently am busy with some personal projects. I will continue to work on this fork and hopefully in the future push it into Zone117x repo in the future.

If you need help, please provide a better question on what is not working and any errors you receive.

**Comment by shtse8** [comment]
Yes. I have successfully installed. My main concern is we want to choose an active project to use, so that we can help developing and keep using for long run. I really hope this NOMP can support Crytonight also soon.

**Comment by krisklosterman** [comment]
I want this to be active, I plan on working on it and welcome anyone else to come work on it. The big thing that needs to be done is testing of other algo's and getting scrypt jane to work.

**Comment by shtse8** [comment]
I have stared and watched this project. Please keep it active and attract more people to join. Cheers.

**Comment by ghost** [comment]
Hashing algorithms supported:

✓ SHA256 (tested with VCOIN)
✓ Scrypt (tested with AntiLiteCoin, 1CREDIT, ArgusCoin, WAYAWOLFCOIN and many others)
(-) Scrypt-Jane (submitblock not working tested with CacheCoin, Yacoin)
? Scrypt-N (need tests)
? Quark (need tests)
✓ X11 (tested with BrainCoin, CannabisCoin, AdzCoin and many others)
? X11Ghost (need tests)
✓ Groestl (tested only shares with AuroraCoin, blocks not tested)
? X13 (need tests)
? Lyra2Z (need tests)
? NIST5 (need tests)
? Keccak (need tests)
? Skein (need tests)
? Blake (need tests)
? Fugue (need tests)
? Qubit (Shares works, blocks finding, but error 'We thought a block was found but it was rejected by the daemon'. No blocks presented in pools web interface)
? SHAvite-3 (need tests)
? Sha1 (need tests)
? Hefty1 (need tests)
? Cryptonight (need tests)

**Comment by krisklosterman** [comment]
#6
